includes/wikia/GlobalFunctions.php

```diff
@@ -1609,12 +1609,15 @@ function wfGetEffectiveHostname() {
 }
 
 function wfGetBaseDomainForHost( $host ) {
-	global $wgWikiaBaseDomain, $wgFandomBaseDomain;
+	global $wgWikiaBaseDomain, $wgWikiaOrgBaseDomain, $wgFandomBaseDomain;
 	if ( strpos( $host, ".{$wgFandomBaseDomain}" ) !== false ) {
 		return $wgFandomBaseDomain;
 	}
+	if ( strpos( $host, ".{$wgWikiaBaseDomain}" ) !== false ) {
+		return $wgWikiaBaseDomain;
+	}
 
-	return $wgWikiaBaseDomain;
+	return $wgWikiaOrgBaseDomain;
```

Comment:
Since most wikis are no on fandom.com, I'd make `wgFandomBaseDomain` the default return now, rather than `wikia.org`.

Reply:
@Grunny it seems that this change breaks one of the tests. The default behavior should be that sitemap links for other domains should be generated with the http address extensions/wikia/RobotsTxt/tests/WikiaRobotsTest.php:testSitemap() . Do we want to change it or should it stay the way it is?

Reply:
Leaving it as `wgWikiaBaseDomain` is probably OK, but just changing the test to `https` may make more sense since long-term we won't have anything left on `wikia.com`, my main concern here was `wgWikiaOrgBaseDomain` being the default (and `wikia.org` will also be HTTPS-only anyway). So, I'll leave it up to you. :)